src/resources/authoring/gv2.json

```diff
@@ -1,397 +1,424 @@
+        "mother":{
+          "alleles": "w-W, M-m, fl-, hl-, T-T, h-h, C-C, A1-A1, B-B, D-D, rh-rh, Bog-Bog",
           "sex": 1
         },
-        {
-          "alleles": "A1-a, H-h, Bog-, D-, rh-rh",
+        "father": {
+          "alleles": "w-W, m-m, T-T, h-h, A1-A1, C-C, B-B, D-D, rh-rh, Bog-Bog",
           "sex": 0
         },
-        {
-          "alleles": "a-A1, h-h, Bog-, D-, rh-rh",
-          "sex": 1
-        }
-      ],
-      "targetDrakes": [
-        {
-          "alleles": "A1-A1, h-h",
+        "visibleGenes": "metallic, wings, forelimbs, hindlimbs"
+      },
+      {
+        "comment": "*** Dragon 3.1: Draco (piece 2): #5/2 - Egg Game I(g) (select gametes to create five unique drakes) ***",
+        "template": "EggGame",
+        "challengeType": "create-unique",
+        "interactionType": "select-gametes",
+        "instructions": "Create 5 different baby drakes!",
+        "showUserDrake": true,
+        "mother":{
+          "alleles": "w-W, M-m, fl-, hl-, T-T, h-h, C-C, A1-A1, B-B, D-D, rh-rh, Bog-Bog",
           "sex": 1
         },
-        {
-          "alleles": "A1-A1, h-h",
+        "father": {
+          "alleles": "w-W, m-m, T-T, h-h, A1-A1, C-C, B-B, D-D, rh-rh, Bog-Bog",
+          "sex": 0
+        },
+        "gameteCounts": [8, 8],
+        "visibleGenes": "metallic, wings, forelimbs, hindlimbs"
+      },
+      {
+        "comment": "*** Dragon 3.1: Draco (piece 3): #5/3 - Egg Game II(g) (select gametes for breeding to match drakes) ***",
+        "template": "EggGame",
+        "challengeType": "match-target",
+        "interactionType": "select-gametes",
+        "instructions": "Match the target drakes!",
+        "showUserDrake": true,
+        "mother":{
+          "alleles": "w-W, M-m, fl-, hl-, T-T, h-h, C-C, A1-A1, B-B, D-D, rh-rh, Bog-Bog",
           "sex": 1
         },
-        {
-          "alleles": "a-a, H-h",
+        "father": {
+          "alleles": "w-W, m-m, T-T, h-h, A1-A1, C-C, B-B, D-D, rh-rh, Bog-Bog",
           "sex": 0
-        }
-      ],
-      "userChangeableGenes": "armor, horns",
-      "hiddenAlleles": "A2",
-      "linkedGenes": {
-        "drakes": [0, 1],
-        "genes": "metallic, wings, forelimbs, hindlimbs, tail, color, black, dilute, nose, bogbreath"
+        },
+        "gameteCounts": [[2, 1], [1, 3], [4, 4]],
+        "targetDrakes": [{}, {}, {}],
+        "visibleGenes": "metallic, wings, forelimbs, hindlimbs"
       }
-    },
-    {
-      "comment": "*** Dragon 1.3: Cezar (piece 2): #4/2 - Single Drake (User Drake Hidden) ***",
-      "template": "GenomeChallenge",
-      "randomizeTrials": true,
-      "instructions": "Match the target drake!",
-      "showUserDrake": false,
-      "initialDrake": [
-        {
-          "alleles": "A1-A1, H-H, Bog-, D-, rh-rh",
+    ]
+  ],
+  [
+    [
+      {
+        "comment": "*** LEVEL STAND IN ***",
+        "template": "GenomeChallenge",
+        "instructions": "THIS LEVEL NOT YET COMPLETE",
```

Comment:
I think it would make sense to have a `StubTemplate` (or some other more suitable name) which could just put up a message that included the comment and a "Not Yet Implemented" message. With that in place, we wouldn't need to redo the number-wrapping logic.